libcodechecker/analyze/analyzer_env.py

```diff
@@ -95,6 +96,14 @@ def extend_analyzer_cmd_with_resource_dir(analyzer_cmd, compiler_resource_dir):
     and describes our intentions more cleanly if we just simply disable the
     inludes from the resource dir.
     """
-    if len(compiler_resource_dir) > 0:
-        analyzer_cmd.extend(['-nobuiltininc',
-                             '-isystem', compiler_resource_dir])
+    if len(compiler_resource_dir) == 0:
+        return
+
+    resource_inc = compiler_resource_dir
+    # Resource include path must end with "/include".
+    basename = os.path.basename(os.path.normpath(resource_inc))
```

Comment:
Maybe better to move this logic to `__get_compiler_resource_dir`?

Reply:
This is not resolved.

Reply:
On a long run we plan to use a function which takes both the system includes and the resource dir as a parameter, so we could implement just in one place the madness around these headers.